Readme Update (Updates Contributing, Usage) Sections (#228)
iTwin/changed-elements-react: packages/changed-elements-react/README.md

Reviewer: Under "Installation", I suggest changing `npm` to `pnpm` since that's what this project uses. Though technically it depends on the package manager being used by the user.

Reviewer: Where you say to add the OAuth scopes on line 17, perhaps add a link to the developer portal apps page?

    });
    ```

> 4. `<ChangedElementsWidget />` React component lets users inspect differences in properties between versions, generate reports, search for changed elements, and control element visibility. The following code has example of widget use but does not show off all props for widget.

Reviewer: ```suggestion
4. The `<ChangedElementsWidget />` React component lets users inspect differences in properties between versions, generate reports, search for changed elements, and control element visibility. The following code shows an example widget initialization (does not show off all props).
```

    ```ts
    import { ChangedElementsWidget } from "@itwin/changed-elements-react";
    <ChangedElementsWidget
          useV2Widget
          feedbackUrl="https://example.com"

>           iModelConnection={UiFramework.getIModelConnection()!}

Reviewer: The `!` is required here? That feels like bad practice to include in the example, but I do not know enough to say myself.

          enableComparisonJobUpdateToasts
          manageNamedVersionsSlot={<ManageNamedVersions />}
          documentationHref="https://example.com"
        />,
    ```

> 5. `<NamedVersionSelectorWidget />` **Experimental** React component lets users inspect differences in properties between versions, generate reports, search for changed elements, and control element visibility. The following code has example of widget use but does not show off all props for widget.

Reviewer: Here is a suggested change, but also, should this say the same thing as the bullet point above? It's doing the same stuff?
```suggestion
5. The `<NamedVersionSelectorWidget />` is an **experimental** React component that lets users inspect differences in properties between versions, generate reports, search for changed elements, and control element visibility. The following code shows an example widget initialization (does not show off all props).
```

    ```ts
    import { NamedVersionSelectorWidget } from "@itwin/changed-elements-react";
    <NamedVersionSelectorWidget
      iModel={iModel}
      manager={VersionCompare.manager}
      manageVersions={<ManageNamedVersions />}
      feedbackUrl="https://example.com"
      documentationHref="https://example.com"

>       />

Reviewer: You have inconsistent tab-spacing in each of your examples. Make them all consistent.

    ```

> 6. `<PropertyComparisonTable />` React component lists properties of a selected element and displays how they changed between two versions.

Reviewer: ```suggestion
6. The `<PropertyComparisonTable />` React component lists properties of a selected element and displays how they changed between two versions.
```
